algradigon-1/upgrades/upgrades-using-cosmosd.md

```diff
@@ -126,10 +126,16 @@ sudo systemctl stop xrnd
 
 ```
 cd $DAEMON_HOME/upgrade_manager
-mkdir -p upgrades/papua/bin
-cd upgrades/papua
+mkdir -p upgrades/patagonia/bin
+cd upgrades/patagonia
 curl -L -o bin/xrnd https://github.com/regen-network/regen-ledger/releases/download/v0.5.1/xrnd-v0.5.1
```

Comment:
Now I think the only thing is to make this link to the actual release asset is that right @anilCSE ?

Reply:
Ah looks like this is the name of the release binary. Would be good to also provide a `.tar.xz` like the `regen-ledger-v0.5.0-linux-amd64.tar.xz` which we had for amazonas. But this works for now.

Reply:
And maybe checksums published somewhere